src/internal/cache-client.ts

```diff
@@ -1640,6 +1643,172 @@ export class CacheClient {
     });
   }
 
+  public async sortedSetPutValue(
```

Comment:
for similarity with Sets we may wanna go with Element here.

Reply:
Fixed in https://github.com/momentohq/client-sdk-nodejs/pull/314/commits/3d63768766c96de67c2e103e8705369a6b28a6c2